src/sentry/workflow_engine/endpoints/project_detector_index.py

```diff
@@ -0,0 +1,71 @@
+from rest_framework import status
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+
+from sentry.api.bases import ProjectAlertRulePermission, ProjectEndpoint
+from sentry.api.serializers import serialize
+from sentry.issues import grouptype
+from sentry.workflow_engine.models import Detector
+
+
+class ProjectDetectorIndexEndpoint(ProjectEndpoint):
```

Comment:
should we also mark these APIs as experimental?

Reply:
I think I need to add docs in general here to make ci pass, so I'll do that while I'm adding those